src/game/game.cpp

```diff
@@ -5694,16 +5694,18 @@ void Game::playerChangeOutfit(uint32_t playerId, Outfit_t outfit, uint8_t isMoun
 			outfit.lookMount = 0;
```

Comment:
eu substitui esse if `if (playerTile->hasFlag(TILESTATE_PROTECTIONZONE)) {` por e removi o código abaixo, e está funcionando.
```suggestion
			if (playerTile->hasFlag(TILESTATE_PROTECTIONZONE) && !g_configManager().getBoolean(TOGGLE_MOUNT_IN_PZ, __FUNCTION__)) {
			outfit.lookMount = 0;
		} else {
			player->toggleMount(true);
		}
```

Reply:
@elsongabriel pode abrir um pull request a parte? assim eu peço para a pessoa que me reportou o bug, testar.